package-scripts.js

```diff
@@ -54,13 +42,15 @@ module.exports = {
       description: 'auto-formats the code',
       script: 'prettier-eslint --write "src/**/*.js"',
     },
-    addContributor: {
```

Comment:
Awesome! I definitely prefer the new style. Could you update the `CONTRIBUTING.md` references to these scripts?